invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -86,22 +88,28 @@ if (sidebarCommunitiesManageDiv) {
   const pendingCommunitiesSearchConfig =
     sidebarCommunitiesManageDiv.dataset.pendingCommunitiesSearchConfig;
   ReactDOM.render(
-    <CommunitiesManagement
-      userCommunitiesMemberships={JSON.parse(
-        sidebarCommunitiesManageDiv.dataset.userCommunitiesMemberships
-      )}
-      recordCommunityEndpoint={
-        sidebarCommunitiesManageDiv.dataset.recordCommunityEndpoint
-      }
-      recordUserCommunitySearchConfig={JSON.parse(
-        sidebarCommunitiesManageDiv.dataset.recordUserCommunitySearchConfig
-      )}
-      canManageRecord={JSON.parse(sidebarCommunitiesManageDiv.dataset.canManageRecord)}
-      recordCommunitySearchConfig={recordCommunitySearchConfig}
-      permissions={JSON.parse(sidebarCommunitiesManageDiv.dataset.permissions)}
-      searchConfig={JSON.parse(pendingCommunitiesSearchConfig)}
-      record={JSON.parse(recordCitationAppDiv.dataset.record)}
-    />,
+    <OverridableContext.Provider value={overriddenComponents}>
+      <Overridable id="InvenioAppRdm.RecordLandingPage.CommunitiesManagement.container">
```

Comment:
the props which should be passed to overridable component are missing. Basically all the props you are using in CommunitiesManagement won't be available in your new local component, and there is a high possibility you will need them, since they are taking care of main functionality

Reply:
I Provided all the props to the Overridable.